include/wx/private/lang_likely.h

```diff
@@ -0,0 +1,353 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/private/lang_likely.h
+// Purpose:     Mapping of likely subtags
+// Author:      Vadim Zeitlin
+// Created:     2024-10-04
+// Copyright:   (c) 1998 Vadim Zeitlin <zeitlin@dptmaths.ens-cachan.fr>
```

Comment:
Sorry, I have nothing useful, just two nitpicks:
1. I think you should be listed as "Author" instead of Vadim (and according to the coding guidelines, your e-mail should be there in angle brackets).
2. "Copyright" should be provided as "(c) 2024 wxWidgets development team" (and Vadim's e-mail here is probably quite outdated, vadim@wxwidgets.org is being used elsewhere).

The same applies to other new files.

Reply:
> Sorry, I have nothing useful, just two nitpicks:
> 
> 1. I think you should be listed as "Author" instead of Vadim (and according to the coding guidelines, your e-mail should be there in angle brackets).

Well, these files are actually generated by the script misc/languages/genlang.py. Therefore I set the script as the author.

> 2. "Copyright" should be provided as "(c) 2024 wxWidgets development team" (and Vadim's e-mail here is probably quite outdated, [vadim@wxwidgets.org](mailto:vadim@wxwidgets.org) is being used elsewhere).

I adjusted the copyright line of the new files accordingly.

Reply:
> I adjusted the copyright line of the new files accordingly.

I apologize if I was unclear: I don't think Vadim's e-mail address should be included in the "Copyright: " line: It is not done so in any other files with team copyright. I mentioned the e-mail address in case you wanted to retain Vadim's copyright.